source/acquaman/BioXAS/BioXASXASScanActionController.cpp

```diff
@@ -158,6 +174,8 @@ void BioXASXASScanActionController::buildScanControllerImplementation()
 		connect( BioXASBeamline::bioXAS()->i0Detector(), SIGNAL(darkCurrentValueChanged(double)), i0CorrectedDetectorSource, SLOT(setDarkCurrent(double)) );
 
 		scan_->addAnalyzedDataSource(i0CorrectedDetectorSource, true, false);
+
+		exportXDI->addDataSource(i0CorrectedDetectorSource->name(), false);
```

Comment:
Yes. What I'd wanted to achieve here was having the first column in the exported file be the X-axis value for the first data source (in this case the I0 corrected source). There is an exporter option setting for this behavior (`setFirstColumnOnly(bool)`?), but it doesn't appear to apply in cases where `includeAllDataSources()` is false, as it is here.

A better solution might be to add an implementation for this particular case.